aws/table_aws_ec2_key_pair.go

```diff
@@ -78,6 +78,12 @@ func tableAwsEc2KeyPair(_ context.Context) *plugin.Table {
 				Hydrate:     getAwsEc2KeyPairAkas,
 				Transform:   transform.FromValue(),
 			},
+			{
+				Name:        "create_time",
+				Description: "The time that the keypair was created or imported",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("CreateTime"),
```

Comment:
Hi @kasadaamos, Do we need the `Transform` function here? Doesn't the default transformation help us populating the column value?

Reply:
You are right - it works without the extra `Transform`